### PR TITLE
Clarifying that this profile uses First Implementer's drafts of SIOP v2 and OpenID4VP

### PR DESCRIPTION
**Pull request by Sakurann**

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -139,13 +139,17 @@ sequenceDiagram
...
 - To bind an owner of a DID to a controller of a certain origin, a Well Known DID Configuration MUST be used as defined in [[ref: Well Known DID]].
 - For Revocation of VCs, Status List 2021 as defined in [[ref: Status List 2021]] MUST be used in combination with Identity Hubs as defined in [[ref: Identity Hub]] (Decentralized Web Node v0.0.1 predraft).
 
 It is important to note that Cross-device SIOP is susceptible to a session phishing attack, where an attacker relays the request from a good Verifier/RP to a victim and is able to sign in as a victim. Implementers MUST implement mitigations most suitable to the use-case. For more details and concrete mitigations, see section 15 Security Considerations in [[ref: SIOPv2]].
 
+Note: This profile uses First Implementer's Drafts of Self-Issued OpenID Provider v2 and OpenID for Verifiable Prensetations. This document does not necessarily rely on the latest editor's drafts of these specifications but may choose to do so anytime.
```
Comment (dtmcg) [inline spec/spec.md:149]: is there a link to the implementers draft which can be included?

Comment (dtmcg) [inline spec/spec.md:149]: is it `First Implementer's Drafts` or `first Implementer Draft` or just `Implementer's Drafts` with a link ?

Comment (Sakurann) [reply spec/spec.md:149]: it's included in the reference section.........

Comment (Sakurann) [reply spec/spec.md:149]: `This profile uses First Implementer's Drafts of Self-Issued OpenID Provider v2 and OpenID for Verifiable Prensetations specifications.`

**Review by jischr — APPROVED** [review]

**Pull request merged**